src/basic_bot/test_helpers/camera_mock.py

```diff
@@ -62,4 +63,6 @@ def frames() -> Generator[bytes, None, None]:
                 img = not_pet_images[index]
 
             yield img  # type: ignore
-            time.sleep(1 / 60)  # Limit supply to ~60 FPS
+            time.sleep(
+                1 / c.BB_CAMERA_FPS - (time.time() - tstart)
```

Comment:
there is still enough slop in here (the time to call time.time()) that matching exactly BB_CAMERA_FPS is hard.  But this allows us to get within a 0.1 sec TOLERANCE below.